### ISSUE TITLE
2018 Burnet precinct value not correct?

### ISSUE THREAD

**Issue by crass** [openelections/openelections-data-tx#277]
Currently the file `2018/20181106__tx__general__burnet__precinct.csv` has a `precinct` field that seems to be the fields `Precinct_name` and `Split_name` appended together.  I believe that should be the `precinct_name` field and the `precinct` field should correspond to the `Pct_Id` field in the source file.

**Comment by dwillis** [comment]
@crass Looking at the source file, I can see the case for this.

**Comment by dwillis** [comment]
@crass Actually, I'm gonna reverse course and say we won't change it. Here's why: doing as you recommend would result in Burnet having greater than 20 precincts, [which it does not](http://www.burnetcountytexas.org/upload/page/0146/docs/VOTING%20PRECINCTS%20-%202019.pdf). Using `Pct_Id` as the precinct would not correspond to the official precinct map. It appears to me that the precincts that use `Split_name` are split precincts and we try to preserve those.

**Comment by crass** [comment]
TL;DR: I don't recommend doing what this issue says, but instead adding the `Split_name` field from the source file.

Upon reflection, I think I am unsure what `precinct` means for our purposes ([precincts](https://www.sos.state.tx.us/elections/forms/webinars/Precincts-and-Polling-Places-and-Redistricting.ppsx) are [confusing](https://statewidedatabase.org/diagrams.html)).  Do we have a working definition of `precinct`?  

From what I gather for Burnet County, some precincts have been subdivided to account for the overlapping of other administrative districts (in this case Marble Falls ISD and Horseshoe Bay city limits) which would modify the ballots for voters in those districts (eg. Prop A for Marble Falls ISD).  I've totaled the US House votes across "precincts" from the source file, and they add up to the [official results](http://www.burnetcountytexas.org/upload/page/0154/docs/2018nov%20burnet%20county%20official%20cumulative.pdf) from the county website.  That means that the precincts we call "20" and "20 MFISD" combine to make up the true precinct as shown on this [map](http://www.burnetcountytexas.org/upload/page/0146/docs/VOTING%20PRECINCTS%20-%202019.pdf).  So if the map is how we define precincts, then our value in the precinct field is not correct.

If we correct it (eg. just using the numeric value of the `precinct` field), then we'll have some rows that appear to be duplicates of a precinct, but with different vote totals. In this case, we might want to just aggregate those rows into one, otherwise the user of the data will need to know to do so.  Alternatively, we should add the `Split_name` field of the source file and not aggregate the rows to preserve the granularity of the data and not lose information.  I like the later option, since I don't like losing data.  But this means that someone wanting to use the data to say generate maps of the precinct data will need to do this aggregation themselves.

I do not recommend my initial proposal in the issue description because the `Pct_Id` field does not always correspond to what the county considers a valid precinct number. It seems to be more of some kind of internal id.  As far as I can tell, `Pct_Id` doesn't correspond to what is a legal "Election District" (eg. there must be a polling location with in it), which is I think usually what a precinct is (as noted in powerpoint link and deduced from map above).
